lib/Time/Duration/Concise.pm

```diff
@@ -207,8 +203,9 @@ sub as_concise_string {
     $precision ||= 10;
     my $time_frames         = $self->_duration_array($precision);
     my @concise_time_frames = map {
-        $_ =~ s/\s+//ig;
```

Comment:
`$_` should not be modified in the list according to `perlcritic` policy.
So I added a variable `$t`. But there is a bug in `Dist::Zilla::Plugin::Test::UnusedVars` (https://github.com/doherty/Dist-Zilla-Plugin-Test-UnusedVars/issues/7)

So I disabled this plugin in `dist.ini`